.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,8 +20,9 @@ about: Create a bug report to help us improve
 <!-- If applicable, add screenshots to help explain your problem. -->
 
 **Smartphone Info:**
-|               | |
-|---------------|-|
+
+|               |   |
```

Comment:
unnecessary change.

Reply:
I know, but these are in accordance with official guidelines of github. You can find it here https://help.github.com/en/articles/organizing-information-with-tables